dynamic-content/tsv/content.tsv

```diff
@@ -158,72 +158,72 @@ Papua New Guinea	APAC			N/A	None								"<br>If you’re asked to provide a cert
 Ukraine	EAN			N/A	None											
-United Arab Emirates	MENASA		<br>To have a religious ceremony in a church or Sharia court you or your partner must be a resident in the United Arab Emirates. <br>To have a civil ceremony at the Dubai Courts Civil Centre or the Abu Dhabi Judicial Department neither you or your partner has to be a resident in the United Arab Emirates.	FALSE	Affirmation	3 months		*if you are planning a religious ceremony and you are not a resident in UAE you will need to provide your partner's proof of UAE residency	FALSE							"<br>Once you’ve applied online, download either the [affirmation](https://www.gov.uk/government/publications/affidavitaffirmatin-of-marital-status-forms-uae) (if you want to make a non-religious declaration) or [affidavit](https://www.gov.uk/government/publications/affidavitaffirmatin-of-marital-status-forms-uae) (religious) and type in your details. Do not fill in by hand or sign it. Email a copy of your form to either [British Embassy Dubai](mailto:consularappointment.uae@fcdo.gov.uk) or [British Embassy Abu Dhabi ](abudhabi.consularappointments@fcdo.gov.uk). They will process your application and confirm your appointment. If you do not email a copy of the completed template your appointment will be cancelled."
+United Arab Emirates	MENASA		<br>To have a religious ceremony in a church or Sharia court you or your partner must be a resident in the United Arab Emirates. <br>To have a civil ceremony at the Dubai Courts Civil Centre or the Abu Dhabi Judicial Department neither you or your partner has to be a resident in the United Arab Emirates.	FALSE	Affirmation	3 months		*if you are planning a religious ceremony and you are not a resident in UAE you will need to provide your partner's proof of UAE residency	FALSE							"<br>Once you’ve applied online, download either the [affirmation](https://www.gov.uk/government/publications/affidavitaffirmatin-of-marital-status-forms-uae) (if you want to make a non-religious declaration) or [affidavit](https://www.gov.uk/government/publications/affidavitaffirmatin-of-marital-status-forms-uae) (religious) and type in your details. Do not fill in by hand or sign it. Email a copy of your form to either [British Embassy Dubai](mailto:consularappointment.uae@fcdo.gov.uk) or [British Embassy Abu Dhabi ](mailto:abudhabi.consularappointments@fcdo.gov.uk). They will process your application and confirm your appointment. If you do not email a copy of the completed template your appointment will be cancelled."

```

Comment:
Is there meant to be a space before the square bracket?

`[British Embassy Abu Dhabi ](mailto:abudhabi.consularappointments@fcdo.gov.uk)`